src/main/scala/com/target/data_validator/stats/SecondPassStatsAggregator.scala

```diff
@@ -0,0 +1,107 @@
+package com.target.data_validator.stats
+
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.expressions.{MutableAggregationBuffer, UserDefinedAggregateFunction}
+import org.apache.spark.sql.types._
+
+/**
+  * Calculate the standard deviation and histogram of a numeric column
+  */
+class SecondPassStatsAggregator(firstPassStats: FirstPassStats) extends UserDefinedAggregateFunction {
+
+  val NUMBER_OF_BINS = 10
```

Comment:
Does it make sense to make this configurable?
```suggestion
  final val NUMBER_OF_BINS = 10
```